repose-aggregator/components/filters/header-normalization-filter/src/main/scala/org/openrepose/filters/headernormalization/HeaderNormalizationFilter.scala

```diff
@@ -71,22 +73,42 @@ class HeaderNormalizationFilter @Inject()(configurationService: ConfigurationSer
 
     val wrappedRequest = new HttpServletRequestWrapper(servletRequest.asInstanceOf[HttpServletRequest])
 
-    config find { target =>
+    configRequest find { target =>
       // find the first "target" config element that matches this request (if any)
       target.url.matcher(wrappedRequest.getRequestURI).matches &&
```

Comment:
I'm not sure we would want to in this case. As it is, the `Pattern` is only compiled to a `Matcher` one time when the config is updated. The `RegexStringOperators` `trait` doesn't look like caches the `Matcher` unless I'm missing something. So with the `trait` it would compile it every time it tried to match.